app/src/main/java/com/screen/MainActivity.java

```diff
@@ -0,0 +1,41 @@
+package com.screen;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
+import com.framgia.music_45.R;
+import com.screen.home.FragmentGenres;
+
+public class MainActivity extends AppCompatActivity
+        implements BottomNavigationView.OnNavigationItemSelectedListener {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        BottomNavigationView navigationView = findViewById(R.id.bottom_navigation);
+        addFragment();
+        navigationView.setOnNavigationItemSelectedListener(this);
+    }
+
+    @Override
+    public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+        switch (menuItem.getItemId()) {
+            case R.id.bottomnavigationHome:
+                addFragment();
+        }
+        return false;
+    }
+
+    private void addFragment() {
+        FragmentGenres Genres = FragmentGenres.newInstance();
```

Comment:
xoá dòng này và a edit lại dong 38 như thế này 
```suggestion
       fragmentTransaction.add(R.id.frame, FragmentGenres.newInstance());
```